### ISSUE TITLE
[research] analysis on swap fee rate and pool investor profit

### ISSUE THREAD

**Issue by dlguddus** [tendermint/liquidity#82]
Comparing risk-return profile of pool investors between Uniswap and Liquidity Module

- Risk
  - Impermenant Loss
    - impermenant loss is caused by reserve token balance changes until withdraw
    - this risk is exactly same both in Uniswap and Liquidity Module
  - Slippage Return
    - in Uniswap, there exists no mechanism to protect liquidity pool from impermenant loss by huge instant price changes
    - but, in Liquidity Module, there exists a mechanism to partially protect liquidity pool from significant impermenant loss
      - huge instant price changes lead to longer batch period
      - which attracts more arbitrageurs, resulting in swap price near global price
      - then, assuming perfect competition among arbitrageurs, impermenant loss can be partially covered
      - by the swap price which is near global price
    - path dependency : this protection mechanism effect is minimized if price change for each batch is minimized

</br>

- Return
  - Swap Fee
    - each executed swap pays swap fee proportional to swap amount
    - for swap fees from pool-match, both Uniswap and Liquidity Module has same potential fee returns
    - but, in Liquidity Module, it additionally has swaps which do not spend liquidity from liquidity pool (self-match)
    - therefore, Liquidity Module has additional fee source compared to Uniswap for given liquidity amount
    - this can be described as higher capital efficiency of liquidity pool

</br>

in short, Liquidity Module pool investors have **lower impermenant loss risk**, and **higher swap fee return** than Uniswap

**Comment by dlguddus** [comment]
this topic has been researched and presented from economic simulation report
